framework/src/types.ts

```diff
@@ -153,3 +160,21 @@ export interface TransactionJSON {
 	readonly id: string;
 	readonly asset: object;
 }
+
+// TODO: replace definition from lisk-chain after #5609 "Update lisk-chain to support the on-chain architecture"
+export interface Validator {
```

Comment:
We have `LiskValidator` in validator lib that is used across framework serving completely different purpose to what this `Validator` means, maybe for more clarity and distinction we can rename it to `BlockValidator`?